setup.py

```diff
@@ -0,0 +1,22 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="ngshare",
+    version="0.0.1",
```

Comment:
I usually keep a `ngshare/version.py` that only contains a single variable `__version__` containing a string of the version number. Then here you actually `exec(open('ngshare/version.py').read())` to make it available in setup.py. This gives a single source for the version number (can be used in sphinx docs too). You'll need to `from version import __version__` in your main `__init__.py` also, so `from ngshare import __version__` works.

Reply:
I will look into the setup.py configuration later, but I am a little bit concerned whether it is a good practice to use `exec` or `eval`. When learning Python I was told that it is dangerous.

Reply:
In general you should be very careful using exec and eval, but this is one case where I do use it. There is a chicken and egg problem if you want to store the version number in one location for use in the imported package and during installation, i.e. you need to run setup.py before ngshare is installed. This is a common practice in python setup.py files at this point and I use it on all of the 20+ packages I maintain.

Reply:
See the official recommendations here: https://packaging.python.org/guides/single-sourcing-package-version/

Reply:
I am using the first solution (`get_version` function). It should work in most cases (as long as the version name is not a strange string).